src/hotspot/share/compiler/oopMap.hpp

```diff
@@ -374,7 +374,7 @@ class OopMapStream : public StackObj {
   void next()                           { find_next(); }
   OopMapValue current()                 { return _omv; }
 #ifdef ASSERT
-  int stream_position() const           { return _stream.position(); }
+  int stream_position()                 { return _stream.position(); }
```

Comment:
This change is an example that something is wrong with the design.
There is a concrete class `CompressedReadStream` with expected behaviour of `position`: no changes to `_stream`.  We have to break this contract to be able to compile `OopMapStream`.

Reply:
This change is unrolled as CompressedSparseDataReadStream is a separate class now (rework to avoid the virtual functions)